test/metabase/actions/test_util.clj

```diff
@@ -124,7 +145,7 @@
             (testing "delete row"
               (is (= [1]
                      (jdbc/execute! (sql-jdbc.conn/db->pooled-connection-spec (data/id))
-                                    "DELETE FROM CATEGORIES WHERE ID = 1;"))))
+                                    (delete-categories-1-query)))))
```

Comment:
This test previously wasn't running, because this namespace didn't end in `-test`; now that it's enabled, it was breaking in MySQL because

```
 delete row
expected: (=
           [1]
           (jdbc/execute!
            (sql-jdbc.conn/db->pooled-connection-spec (data/id))
            "DELETE FROM CATEGORIES WHERE ID = 1;"))
  actual: 
  java.sql.SQLSyntaxErrorException: (conn=226) Table 'actions-test-data.CATEGORIES' doesn't exist
```

So it never actually worked in the first place for MySQL. Fixed by making sure we generate a quoted query appropriate for the current driver